examples/sample-app-ts/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "sample-app",
```

Comment:
can you make this `sample-app-typescript`, we use the package names for some things and I think there is a potential for breakage if we have duplicates

Reply:
Done!